dmoj/generator.py

```diff
@@ -4,6 +4,6 @@
 
 
 class GeneratorManager:
-    def get_generator(self, filenames, flags, lang=None, compiler_time_limit=None):
+    def get_generator(self, filenames, flags, lang=None, compiler_time_limit=None, cached=True):
         filenames = list(map(os.path.abspath, filenames))
-        return compile_with_auxiliary_files(filenames, lang, compiler_time_limit)
+        return compile_with_auxiliary_files(filenames, flags, lang, compiler_time_limit, cached)
```

Comment:
Try to keep parameters that are passed together in the same relative location. bridged.py sends in `flags, cached`, but this sends `flags, ..., cached`

Reply:
Now passed in same order as `compile_with_auxiliary_files`.